docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2910,6 +2910,479 @@ have not been included. These are:
 #. We sample 100% of events. This is likely unrealistic, but the percent is probably very high still. 
 #. There are errors in SSN data, which are not modeled here. 
 
+Noise Functions
+^^^^^^^^^^^^^^^
+
+In order to have a realistic challenge with PRL, it is essential to have 
+noise added to the data. We currently divide noise into two types: 
+
+#. **Column based noise:** errors in individual data entry, such as miswriting or incorrectly selecting responses 
+#. **Row based noise:** errors in the inclusion or exclusion of entire rows of data, such as duplication or omission 
+
+Column Noise
+''''''''''''
+
+To begin, we will start with defining column based noise. Some general rules 
+for all column based noise include: 
+
+- Rows eligible for errors is the probability of selecting a row to have a particular type of noise added. For example, 1% noise level for incorrect selection to type of tax form means 1% of rows will be selected to have the wrong value selected. This is somewhat more complicated for: OCR, phonetic, typographic, and numeric miswriting which is elaborated on below. 
+- Token noise level is a noise parameter that only applies to certian noise types and defines the amount of errors expected once a string is selected for noise. This parameter is also elaborated on below. 
+- The amount of each type of noise is individually configurable for each column in each observer. This means that the end user can select that: in the census sex has 2% incorrect selections. The minimum noise is 0% and the maximum if 5% for all column noise. For errors per row, the minimum is 0 and maximum is 5. 
+- Simulants are selected for noise at random. This is true for each type of noise, each column, and each observer. Selection is not based on any attributes and simulants do not have a higher or lower propensity for noise that would carry with them (e.g., there are not "messy" simulants who are more likely to make errors on all fields/forms). 
+- As noise functions for certain columns are common across observers, the table below is organized by column (e.g., first name). Below the table, there is further information and definition on each noise type. 
+- The order of different noise types should not matter, but will go in the order they are listed in the table below. 
```

Comment:
I'm not sure it will make much difference for PRL, especially if the error rates are small, but I think there is often a sort of logical order to apply the noise in. For example, for names in the census, I think an order that would make sense would be

1. nicknames
2. fake names
3. phonetic noise
4. OCR noise
5. keyboard noise

E.g., first someone would decide to use a nickname or fake name, then there might be a misspelling because of phonetic confusion, then there might be a keyboard error in typing in the phonetically misspelled name.

Anyway, I think it's fine to have an arbitrary order for the MVM, but we might want to revisit this for the final version.

Reply:
I agree, I was going to make a separate (less giant) PR with ordering of the noise. This also applies to row noise. I will keep this order in mind for that.